sherpa/astro/data.py

```diff
@@ -366,7 +366,8 @@ def html_pha(pha):
                           ('CHANTYPE', 'The channel type'),
                           ('HDUCLAS2', 'Data stored'),
                           ('HDUCLAS3', 'Data format'),
-                          ('HDUCLAS4', 'PHA format')])
+                          ('HDUCLAS4', 'PHA format'),
+                          ('XFLT0001', 'Stokes (polarimetry)')])
```

Comment:
Technically I think that the `XFLTxxxx` keywords are generic - that is you could have one in a non-polarimetry dataset.

Reply:
So I'm not sure what we do here.